job-runner.toml

```diff
@@ -47,12 +47,20 @@ dir = '~/.cache/cockpit-dev/job-runner-logs'
 link = 'http://127.0.0.1:8000/'
 
 [forge]
-driver='github'
+default = 'github'
 
 [forge.github]
+driver = 'github'
 clone-url = 'https://github.com/'
 api-url = 'https://api.github.com/'
 post = true  # whether to post statuses, open issues, etc.
 user-agent = 'job-runner (cockpit-project/bots)'
 # (at least) one of `token` or `post = false` must be set
 # token = 'ghp_XXX'
+
+[forge.codeberg]
```

Comment:
... but we eventually need to get a cockpituous (-ish) user on Codeberg and add a token for that to cockpituous.

Reply:
Probably.  Right now I only plan to use this for "command subjects" which are limited to:
 - trivial api requests for things like "what is the sha of your main branch?" and "what is the content of your `.cockpit-ci/container` file?".  Those are working fine without a token at present
 - `git clone`

I consider a potential topic of discussion if we should have Codeberg configured here or not (ie: or leave it to the user's config), but we configure GitHub, so...